Tasks/AzureFileCopyV1/AzureFileCopyJob.ps1

```diff
@@ -51,17 +51,7 @@ param (
         $blobStorageURI = $blobStorageEndpoint+$containerName+"/"+$blobPrefix
     }
 	
-    [String]$copyToAzureMachinesBlockString = [string]::Empty
-    if([string]::IsNullOrWhiteSpace($additionalArguments))
-    {
-        $copyToAzureMachinesBlockString = "Copy-ToAzureMachines -MachineDnsName `$fqdn -StorageAccountName `$storageAccount -ContainerName `$containerName -SasToken `$sasToken -DestinationPath `$targetPath -Credential `$credential -AzCopyLocation `$azCopyLocation -BlobStorageURI `$blobStorageURI -WinRMPort $winRMPort $cleanTargetPathOption $skipCACheckOption $httpProtocolOption $enableDetailedLoggingOption"
-    }
-    else
-    {
-        $copyToAzureMachinesBlockString = "Copy-ToAzureMachines -MachineDnsName `$fqdn -StorageAccountName `$storageAccount -ContainerName `$containerName -SasToken `$sasToken -DestinationPath `$targetPath -Credential `$credential -AzCopyLocation `$azCopyLocation -AdditionalArguments `$additionalArguments -BlobStorageURI `$blobStorageURI -WinRMPort $winRMPort $cleanTargetPathOption $skipCACheckOption $httpProtocolOption $enableDetailedLoggingOption"
-    }
-    [scriptblock]$copyToAzureMachinesBlock = [scriptblock]::Create($copyToAzureMachinesBlockString)
+    $arguments = Protect-ScriptArguments -InputArgs $additionalArguments
 
-    $copyResponse = Invoke-Command -ScriptBlock $copyToAzureMachinesBlock
-    Write-Output $copyResponse
```

Comment:
Let's check if new version prints output

Reply:
The result would be the same. With direct call, the call's stream would be returned directly to the host.
With Invoke-Expression we would write the stream to a variable, but then it would be returned to the host by **Write-Output**

**For example:**
```pwsh
function Copy-FilesToAzureBlob {
    Write-Host "Copying files to Azure Blob"
}

Copy-FilesToAzureBlob
```

**And**
```pwsh
function Copy-FilesToAzureBlob {
    Write-Host "Copying files to Azure Blob"
}

$copyToAzureBlobString = "Copy-FilesToAzureBlob"

[scriptblock]$copyToAzureBlobStringBlock = [scriptblock]::Create($copyToAzureBlobString)

$copyResponse = Invoke-Command -ScriptBlock $copyToAzureBlobStringBlock
Write-Host $copyResponse
```

The output in both cases: 
`Copying files to Azure Blob`